sinks_test.go

```diff
@@ -166,6 +172,30 @@ func TestSHA256Sum(t *testing.T) {
 	}
 }
 
+func TestSinkSlice(t *testing.T) {
+	t.Parallel()
+
+	want := []string {
+		"testdata/multiple_files/1.txt",
+		"testdata/multiple_files/2.txt",
+		"testdata/multiple_files/3.tar.zip",
+	}
+	got, err := ListFiles("testdata/multiple_files").Slice()
+	if err != nil {
+		t.Error(err)
+	}
+
+	if !reflect.DeepEqual(got, want) {
```

Comment:
I suggest using `go-cmp` here (see `TestSlice` for an example), because the diff output is much better.